rfcs/0023-musl-libc.md

```diff
@@ -0,0 +1,158 @@
+---
+feature: musl-libc
+start-date: 2018-02-19
+author: Will Dietz
+co-authors: (find a buddy later to help our with the RFC)
```

Comment:
I'm concerned that a whole new libc only has one "sponsor" here.

Reply:
I'd happily cosponsor.

Reply:
The more the merrier, I think! This writes down who can be pinged for future musl problems.

Reply:
Would it be a hard task?